chainer/function_hook.py

```diff
@@ -76,15 +84,24 @@ class FunctionHook(object):
        as a thread local object. So, function hooks registered
        are different depending on threads.
 
+    If the hook is registered in this way, :meth:`~chainer.FunctionHook.added`
+    and :meth:`~chainer.FunctionHook.deleted` are given ``None`` as the
+    ``function`` argument.
+
     The other one is to register directly to
     :class:`~chainer.FunctionNode` object with
-    :meth:`~chainer.Function.add_hook` method.
+    :meth:`~chainer.FunctionNode.add_hook` method.
     Function hooks registered in this way can be removed by
-    :meth:`~chainer.Function.delete_hook` method.
+    :meth:`~chainer.FunctionNode.delete_hook` method.
     Contrary to former registration method, function hooks are registered
     only to the function which :meth:`~chainer.FunctionNode.add_hook`
     is called.
 
+    If the hook is registered in this way, the :class:`~chainer.FunctionNode`
+    instance is given as the ``function`` argument of
```

Comment:
Suggestion (similar to the above one): given as -> passed to

Reply:
Fixed using "passed as".